src/test/java/com/switchfully/parkshark_2024_10/user/UserRepositoryTest.java

```diff
@@ -0,0 +1,82 @@
+package com.switchfully.parkshark_2024_10.user;
+
+import com.switchfully.parkshark_2024_10.division.Division;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+public class UserRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Test
+    void findByEmailAndPassword_shouldReturnCorrectPerson() {
+        // given
+        Division division = new Division();
+
+        Director director = new Director(division, "Jane", "Doe", "jane.doe@parkshark.com", "password123");
+        userRepository.save(director);
+
+        // When
+        Person found = userRepository.findByEmailAndPassword("jane.doe@parkshark.com", "password123");
+
+        //then
+        Assertions.assertNotNull(found, "Person should not be null");
+        Assertions.assertEquals(director.getId(), found.getId(), "Person IDs should match");
+        Assertions.assertEquals("Jane", found.getFirst_name(), "First names should match");
```

Comment:
Technically this isn't part of this PR, but the method naming is inconsistent with the rest of the codebase. Please change `getFirst_name` to `getFirstName` and `getLast_name` to `getLastName`.